www/hanp05/SP1/styles/styles.css

```diff
@@ -0,0 +1,225 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    border: 0;
+    padding: 0;
+    font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+}
+
+p {
+    margin: 15px;
+}
+
+h1 {
+    font-size: 20px;
+    text-transform: uppercase;
+}
+
+h2 {
+    font-weight: 700;
+    font-size: 18px;
+    text-transform: uppercase;
+    padding-top: 50px;
+    padding-bottom: 10px;
+}
+
+h3 {
+    font-weight: 700;
+    font-size: 16px;
+    text-transform: uppercase;
+    padding-top: 30px;
+    padding-bottom: 20px;
+}
+
+a {
+    text-decoration: none;
+    color: red;
+}
+
+a:hover {
+    color: rgb(255, 106, 106);
+}
+
+.container {
+    position: relative;
+    width: 80%;
+    max-width: 1300px;
+    height: 100%;
+    margin: 0 auto;
+}
+
+.section {
+    width: 100%;
+    background-color: #fff;
+    padding: 60px 0;
+}
+
+/* ****** Header ****** */
+
+.container_header {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    max-width: 1300px;
+    margin: 0 auto;
+    height: 100%;
+}
+
+header, footer {
+    display: grid;
+    align-items: center;
+    width: 100%;
+    background-color: #333;
+    min-height: 80px;
+}
+
+.main_logo {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+}
+
+.book_name {
+    color: rgb(255, 255, 255);
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 20px;
+    white-space: nowrap;
+    margin-right: 10px;
+}
+
+.author_name {
+    color: rgb(153, 153, 153);
+    text-transform: uppercase;
+    font-weight: 100;
+    font-size: 20px;
+    white-space: nowrap;
+}
+
+.menu {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-column-gap: 10px;
+}
+
+.menu a {
+    text-decoration: none;
+    color: white;
+    white-space: nowrap;
+    min-width: 130px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.menu a:hover {
+    background-color: rgba(0, 0, 0, 0.1);
+}
+
+.active_menu_item {
+    background-color: rgba(0, 0, 0, 0.2);
+}
+
+/* ###### Konec Header ###### */
+
+/* ****** Footer ****** */
+
+.footer_image {
+    width: 100%;
+    background-color: rgb(230, 230, 230);
+    padding: 70px 0;
+    background: linear-gradient( rgba(0, 0, 0, 0.55), rgba(0, 0, 0, 0.55)), url('/img/money.jpg');
```

Comment:
money.jpg not found, pouzivej relativni cestu k souborum, navic tenhle obrazek neni optimalizovany, nevim co ostatni obrazky